src/Tablebot/Plugins/Roll/Plugin.hs

```diff
@@ -221,7 +225,7 @@ statsHelp =
     "stats"
     []
     "calculate and display statistics for expressions."
-    "**Roll Stats**\nCan be used to display statistics for expressions of dice.\n\n*Usage:* `roll stats 2d20kh1`, `roll stats 4d6rr=1dl1+5`, `roll stats 3d6dl1+6 4d6dl1`"
+    "**Roll Stats**\nCan be used to display statistics for expressions of dice.\nDoes not work with statements.\n\n*Usage:* `roll stats 2d20kh1`, `roll stats 4d6rr=1dl1+5`, `roll stats 3d6dl1+6 4d6dl1`"
```

Comment:
maybe give an example of statements here? e.g. "Does not work with statements (e.g. var ...)"